manifests/config.pp

```diff
@@ -60,6 +61,9 @@
   if $cache_dirs {
     create_resources('squid::cache_dir', $cache_dirs)
   }
+  if $url_rewrite_program {
+    create_resources('squid::url_rewrite_program', $url_rewrite_program)
```

Comment:
since it's `url_rewrite_program` singular, I'm assuming this will be a string? (and therefore create_resources isn't what you'll need).

Reply:
everything done, something else to handle or ready to merge?